### ISSUE TITLE
reporting level when overriding eslint.json with comment

### ISSUE THREAD

**Issue by jraller** [eslint/eslint#641]
In my eslint.json file I have `"no-extend-native": 2,` and in a .js file I have the following at the top:

```
/* global define, document*/
/*eslint no-extend-native:1 */
```

When I use grunt to eslint the project containing that file (LRAP.js) the report is as follows:

```
Running "eslint:target" (eslint) task

app/scripts/LRAP/LRAP.js
  20:1  error  Number prototype is read only, properties should not be added  no-extend-native

app/scripts/sayName.js
  18:17  warning  sayName is defined but never used  no-unused-vars

? 2 problems
```

It is recognizing that both are set at level 1 for warning, but reporting the no-extend-native as a level 2 (error) rather than a level 1 as it was set in the file (warning). Because it sees both as level 1 grunt is able to go on. If I change the comment to 2 I get the following at the end of the report:

```
Warning: Task "eslint:target" failed. Use --force to continue.

Aborted due to warnings.
```

So It is behaving correctly, but the level of the exception is being reported based on the eslint.json rather than the local comment.
